tests/models/abiotic/test_wind.py

```diff
@@ -295,113 +283,97 @@ def test_calculate_wind_above_canopy():
     from virtual_ecosystem.models.abiotic.wind import calculate_wind_above_canopy
 
     result = calculate_wind_above_canopy(
-        friction_velocity=np.array([0.0, 0.819397, 1.423534]),
-        wind_height_above=np.array([[2.0, 32.0, 32.0], [np.nan, 30.0, 30.0]]),
-        zeroplane_displacement=np.array([0.0, 25.312559, 27.58673]),
-        roughness_length_momentum=np.array([0.017, 1.4533, 0.9591]),
-        diabatic_correction_momentum=np.array([0.003, 0.026, 0.013]),
+        friction_velocity=np.array([0.0, 0.819397, 1.423534, 1.423534]),
+        wind_height_above=np.array(
+            [[2.0, 32.0, 32.0, 32.0], [np.nan, 30.0, 30.0, 30.0]]
+        ),
+        zeroplane_displacement=np.array([0.0, 25.312559, 27.58673, 27.58673]),
+        roughness_length_momentum=np.array([0.017, 1.4533, 0.9591, 0.9591]),
+        diabatic_correction_momentum=np.array([0.003, 0.026, 0.013, 0.013]),
         von_karmans_constant=CoreConsts.von_karmans_constant,
         min_wind_speed_above_canopy=0.55,
     )
 
-    exp_result = np.array([[0.55, 3.180068, 5.478385], [np.nan, 2.452148, 3.330154]])
+    exp_result = np.array(
+        [[0.55, 3.180068, 5.478385, 5.478385], [np.nan, 2.452148, 3.330154, 3.330154]]
+    )
     np.testing.assert_allclose(result, exp_result, rtol=1e-3, atol=1e-3)
 
 
-def test_calculate_wind_canopy(dummy_climate_data):
+def test_calculate_wind_canopy(dummy_climate_data_varying_canopy):
     """Test below canopy wind profile."""
 
     from virtual_ecosystem.models.abiotic.wind import calculate_wind_canopy
 
+    # VIVI - different from attenuation coefficient in the data object? Can we switch to
+    # using the fixture?
```

Comment:
yes, I think that is left over from when I sporadically tested the different number of canopy layers, before having a fixture for that